ironsource/api/ironsource.script_api

```diff
@@ -15,6 +15,15 @@
       - name: gdpr_consent
```

Comment:
This parameter will be ignored and may mislead the user

Reply:
Actually it is optional and if set, then set_consent() will be invoked before init().
I've added "optional: true" to this parameter in .script_api